### PR TITLE
Update banners to account for both GitLab and GH App accounts

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Update banners - Contexts CLI and Renaming guides

# Reasons
Support for all standalone projects

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/rename-organizations-and-repositories.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ contentTags:
 :icons: font
 :experimental:
 
-NOTE: The content on this page is only relevant to GitHub and Bitbucket. Due to the standalone nature of GitLab accounts in CircleCI, no changes are required when orgs or repos are renamed.
+NOTE: The content on this page is only relevant to GitHub OAuth app and Bitbucket integrations. Due to the standalone nature of GitLab and GitHub App accounts in CircleCI, no changes are required when orgs or repos are renamed. To find out if you authorized through the GitHub OAuth app or the CircleCI GitHub App, see the xref:github-apps-integration#[GitHub App integration] page.
```
Comment (kbatuigas) [inline jekyll/_cci2/rename-organizations-and-repositories.adoc:13]: I think we don't have the ability to rename orgs or projects for standalone, so perhaps we mention that in here instead?

**Review by kbatuigas — COMMENTED** [review]
Just one clarifying question

File: jekyll/_cci2/rename-organizations-and-repositories.adoc (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/rename-organizations-and-repositories.adoc:13]: So my understanding is these instructions for people who rename orgs and repos in their VCS, so I think the text as is is OK? We could add something about how renaming of CCI orgs and projects isn't currently possible too (but will be in future), is that what you mean?

Comment (kbatuigas) [reply jekyll/_cci2/rename-organizations-and-repositories.adoc:13]: @rosieyohannan Ohh, thanks, I was confused, I thought it was referring to CCI orgs and repos. I can't remember if we mention elsewhere about renaming of CCI orgs and projects but I don't think that needs to also go here.

**Review by kbatuigas — APPROVED** [review]

File: jekyll/_cci2/rename-organizations-and-repositories.adoc (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/rename-organizations-and-repositories.adoc:13]: OK I've just taken a look at the page with fresh 👀 and it really isn't clear is it! I will try to improve

**Pull request merged**